core/src/test/scala/unit/kafka/coordinator/transaction/TransactionMarkerRequestCompletionHandlerTest.scala

```diff
@@ -187,6 +187,12 @@ class TransactionMarkerRequestCompletionHandlerTest {
     verifyRetriesPartitionOnError(Errors.NOT_ENOUGH_REPLICAS_AFTER_APPEND)
   }
 
+  @Test
```

Comment:
Could we add a unit test for broker-side handling of `writeTxnMarker` with replica manager mocked to return smaller magic number?